statistics/handle/handle.go

```diff
@@ -568,6 +575,7 @@ func (h *Handle) SaveStatsToStorage(tableID int64, count int64, isIndex int, hg
 		if err != nil {
 			return
 		}
+		lastAnalyzePos = upperBound.GetBytes()
```

Comment:
We only need `GetBytes` for the last bucket?

Reply:
The value type may not be bytes, we need the converted form.

Reply:
Oh I mean we can only call `GetBytes()` for the last bucket to avoid unnecessary CPU cycles?